bindings/objective-c/librpc/librpc.m

```diff
@@ -306,9 +306,14 @@ @implementation RPCClient {
     rpc_connection_t conn;
 }
 
-- (void)connect:(NSString *)uri {
+- (BOOL)connect:(NSString *)uri {
     client = rpc_client_create([uri UTF8String], NULL);
-    conn = rpc_client_get_connection(client);
+    if (client) {
+        conn = rpc_client_get_connection(client);
+        return YES;
+    } else {
+        return NO;
+    }
 }
```

Comment:
Shouldn't a failed connect throw an exception instead?

Reply:
I need to know if the service is running without crashing. Is there another option for doing that?

Reply:
As we discussed, the best option would be to return NSError * (as output parameter). error cause can be obtained with `rpc_get_last_error()`.

Reply:
👍 I will update the pr